drivers/completer/completer.go

```diff
@@ -322,6 +322,13 @@ func (c completer) complete(previousWords []string, text []rune) [][]rune {
 	}
 	/* XXX: implement tab completion for DELETE ... USING */
 
+	/* Complete CREATE */
+	if tailMatches(IGNORE_CASE, previousWords, "CREATE") || tailMatches(IGNORE_CASE, previousWords, "CREATE", "TEMP|TEMPORARY") {
+		return completeFromList(text, "DATABASE", "SEQUENCE", "TABLE", "VIEW", "TEMPORARY")
```

Comment:
This needs to be split into separate cases, as `TEMP|TEMPORARY` doesn't apply to `DATABASE`, `SEQUENCE` and another `TEMPORARY`.